### ISSUE TITLE
Property `interpolation-mode` appears to be spelled incorrectly

### ISSUE THREAD

**Issue by ncoden** [sasstools/sass-lint#951]
I am using `gulp-sass-lint v1.3.2` with `sass-lint v1.10.2`.

For the following code:
```scss
  img {
    // Get rid of gap under images by making them display: inline-block; by default
    display: inline-block;
    vertical-align: middle;

    // Grid defaults to get images and embeds to work properly
    max-width: 100%;
    height: auto;
    -ms-interpolation-mode: bicubic;
  }
```
(from [foundation-sites / _global.scss](https://github.com/zurb/foundation-sites/blob/546a5b9f38487450df0dcf4957ba8c73bf11194e/scss/_global.scss#L156))

The following warning is throw:
```
163:5   warning  Property `interpolation-mode` appears to be spelled incorrectly  no-misspelled-properties
```

The sass-lint config is :
```yml
...
  no-misspelled-properties:
    - 1
    - extra-properties: []
...
```
(from [.sass-lint.yml](https://gist.github.com/ncoden/ec50bad6cd2f08df0780e9f356442794))

I expect this warning to not be thrown. `-ms-interpolation-mode` is an non-standard property for IE<=8 (see https://developer.mozilla.org/fr/docs/Web/CSS/Image-rendering)

**Edit**: I have the same error for `-webkit-overflow-scrolling`.

**Comment by DanPurdy** [comment]
This is a bug that's actually being worked on as we speak by @bgriffith
